### PR TITLE
Add more content on writing Atmosphere packages

### PR DESCRIPTION
**Pull request by skirunman**
Thanks for submitting a PR! We'll try to look at it as soon as possible.

If you are adding significant new content, please take a moment to include an update to the [changelog](https://github.com/meteor/guide/blob/master/README.md#changelog) in your PR.

### CODE REVIEW

File: content/writing-packages.md [hunk 1]
```diff
@@ -47,6 +49,8 @@ The `package.js` file is the main file in every Meteor package. This is a JavaSc
 
 In this guide article, we will go over some important points for building packages, but we won't explain every part of the `package.js` API. To learn about all of the options, [read about the `package.js` API in the Meteor docs.](http://docs.meteor.com/#/full/packagejs)
 
+Don't forget to run `meteor add [my-package]` once you have finished developing your package in order to use it, even if the package is for internal use only and you don't intend to publish the package to Atmosphere.
```
Comment (stubailo) [inline content/writing-packages.md]: This is specifically for local packages in an application, would be good to clarify that!

Comment (skirunman) [inline content/writing-packages.md]: Not 100% sure what you want clarified, but I tweaked the text. Let me know if this is what you wanted.

**Comment by stubailo** [comment]
Alright, looks good to me!

**Comment by stubailo** [comment]
Merged, thanks! b3324dfa05dd44991656c7a7055b781778a3257d

**Pull request closed without merging**